### DISCUSSION TITLE
Confluence Integration

### DISCUSSION THREAD

**Discussion by MommasLittleHelper** [flowershow/flowershow#499] [General and Help]
Hi!

Was interested if there's an integration with Confluence planned?

Thanks! :)

**Comment by rufuspollock** [comment]
@MommasLittleHelper what kind of integration? What kind of features were you imaginging?

**Comment by nosovk** [comment]
check https://obsidian-confluence.com/
you may use it to sync with confluence

**Comment by rufuspollock** [comment]
Closing this for now. Think we had a partial answer.
